api/py/ai/chronon/scheduler/interfaces/flow.py

```diff
@@ -0,0 +1,28 @@
+class Flow:
```

Comment:
Add a comment on the purpose the `Flow`?

Reply:
Added